lib/utils.js

```diff
@@ -44,7 +44,11 @@ utils.filename = function filename(fp) {
 
 utils.isPath = function isPath(pattern, opts) {
   return function(fp) {
-    return pattern === utils.unixify(fp, opts);
```

Comment:
here, we'll need to add `if (!opts) opts = {}` to ensure the object exists. the tests are currently failing on travis, I think this should fix it.

also, great idea on the check, and thank you!